CHANGELOG

```diff
@@ -17,6 +17,11 @@ The rules for this file:
 
   * 0.2.2
```

Comment:
Make this 0.3.0 – enhancements cannot go into patch level releases (we use semantic versioning!)